{
	"name": "1_separateErrorRowstoOtherTable",
	"properties": {
		"activities": [
			{
				"name": "Data flow1",
				"type": "ExecuteDataFlow",
				"dependsOn": [],
				"policy": {
					"timeout": "0.12:00:00",
					"retry": 0,
					"retryIntervalInSeconds": 30,
					"secureOutput": false,
					"secureInput": false
				},
				"userProperties": [],
				"typeProperties": {
					"dataflow": {
						"referenceName": "ErrorRowsHandleDataFlow",
						"type": "DataFlowReference"
					},
					"compute": {
						"coreCount": 8,
						"computeType": "General"
					},
					"traceLevel": "Fine"
				}
			}
		],
		"folder": {
			"name": "real_time_scenarios"
		},
		"annotations": [],
		"lastPublishTime": "2024-10-23T12:06:09Z"
	},
	"type": "Microsoft.DataFactory/factories/pipelines"
}